packages/lesswrong/lib/voting/votingSystems.tsx

```diff
@@ -245,6 +246,50 @@ registerVotingSystem({
   },
 });
 
+registerVotingSystem({
+  name: "threeAxisEmojis",
```

Comment:
Maybe some stupidly long name would be less likely to collide? `threeAxisEmojisEAAlpha`

Reply:
I'm not sure collision are actually something we need to worry about? Anyone adding a new system can just pick a longer name. It would also make it annoying when it _stops_ being alpha - we'd either need to write a migration to rename it, or create a whole new voting system leaving the old one in the code as legacy.